src/assets/style/variables/resolution.scss

```diff
@@ -0,0 +1,21 @@
+$breakpoints: (
+  "mobile": (
+    min-width: 375px
+  ),
+  "tablet": (
+    min-width: 768px
+  ),
+  "desktop": (
+    min-width: 1200px
+  )
+);
+@mixin res($name) {
+  @if map-has-key($breakpoints, $name) {
+    @media #{inspect(map-get($breakpoints, $name))} {
+      @content;
+    }
+  } @else {
+    @warn "Unfortunately, this value does not exist in resolution.scss `#{$breakpoint}`. "
```

Comment:
I have a question about that warning. What will be steps to reproduce it ? I tried resizing my window and it didn't appear

Reply:
Use it in style and write a wrong $breakpoint name. Than an error appear

Reply:
Could I ask you to provide an example of code?

Reply:
![image](https://user-images.githubusercontent.com/42767180/60604214-76d8b380-9db7-11e9-88c7-226c32b64f50.png)